### PR TITLE
Fixes  #1148, #1150 HomeFragment - Tablet (Landscape & Portrait) (Low-fi)

### LINKED ISSUE #1148: HomeFragment - Tablet (Landscape) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/5434c52d-b32b-4666-8b28-cf03b3cbd4cd/L-Home-Screen

Implement low-fi UI for **HomeFragment** tablet landscape mode

**Target PR date**: 7 June 2020
**Target completion date**: 10 June 2020

### LINKED ISSUE #1150: HomeFragment - Tablet (Portrait) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/622708db-29f8-4ebf-9cca-1a2acaa1f579/P-Home-Screen/

Implement low-fi UI for **HomeFragment** tablet portrait mode

**Target PR date**: 7 June 2020
**Target completion date**: 10 June 2020

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation
- Implemented the low-fi UI of the HomeFragment - Tablet in both landscape and portrait mode
- Fixes #1148 
- Fixes #1150 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt [hunk 1]
```diff
@@ -413,7 +413,11 @@ class HomeActivityTest {
   @Test
   fun testHomeActivity_checkSpanForItem0_spanSizeIsTwo() {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt]: Rename to `testHomeActivity_checkSpanForItem0_spanSizeIsTwoOrThree`

File: app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt [hunk 2]
```diff
@@ -428,7 +432,11 @@ class HomeActivityTest {
   fun testHomeActivity_configurationChange_checkSpanForItem0_spanSizeIsTwo() {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt]: Rename to: `testHomeActivity_configurationChange_checkSpanForItem0_spanSizeIsThreeOrFour`

**Review by rt4914 — APPROVED** [review]
From lowfi point of view this PR LGTM. Thanks.

File: app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt (hunk 2 above)
Comment (MohamedMedhat1998) [reply app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt (hunk 2 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt]: LGTM.

**Pull request merged**